test/parallel/test-fs-read-stream-pos.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+const common = require('../common');
+const tmpdir = require('../common/tmpdir');
+const fs = require('fs');
+const assert = require('assert');
+const path = require('path');
+
+tmpdir.refresh();
+
+const file = path.join(tmpdir.path, '/read_stream_pos_test.txt');
+
+fs.writeFileSync(file, '');
+
+let counter = 0;
+
+setInterval(() => {
+  counter = counter + 1;
+  const line = `hello at ${counter}\n`;
+  fs.writeFileSync(file, line, { flag: 'a' });
+}, 1);
+
+const hwm = 10;
+let bufs = [];
+let isLow = false;
+let cur = 0;
+let stream;
+
+setInterval(() => {
+  if (stream) return;
+
+  stream = fs.createReadStream(file, {
+    highWaterMark: hwm,
+    start: cur
+  });
+  stream.on('data', common.mustCallAtLeast((chunk) => {
```

Comment:
Any reason this is a mustCAllAtLeast and not a mustCall?

Reply:
mustCall is exact times, but this place will call at least 1 time not exact 1 time.